lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -34,10 +37,20 @@ def to_native(self, course_overview):
                 kwargs={'course_id': course_id},
                 request=request
             )
+            try:
+                get_discussion_course_or_404(CourseKey.from_string(course_id), request.user)
```

Comment:
@nasthagiri 
So for now with course_overview we do not have course tabs information.
Do we add it to the course_overview  to avoid a separate course module fetch call?
and if yes; then how long does cache retains course_overview data. Because all cached obhects would not have tabs info if we do add.